src/az390.java

```diff
@@ -419,6 +419,10 @@ public  class  az390 implements Runnable {
 		* 2022-05-10 AFK #398 fix typo in error message
         * 2022-05-12 DSH #325 issue error 54 invalid DC field terminator if not '..' for BDEFHLPXZ
 		* 2022-06-10 DSH z16 #423 correct rotate instruction errors reported by Dan Greiner
+	    * 2022-06-27 DSH z16 #424 add BFP constants INF, NAN, QNAN, SNAN, DMIN
+	    *                             See POP 13 pages 9-5 and 19-4
+		* 2022-07-03 DSH #414 add 30 new z16 instructions also see
```

Comment:
Line ends with "also see". Also see what? Abe: I already reviewed this for Don. Never got an answer. Maybe delete the "also see".

Reply:
trailing text removed as per your suggestion.